web/src/app/components/Products/ProductsContainer.ts

```diff
@@ -0,0 +1,17 @@
+import container from 'container';
+import Products from './Products';
+import { Book } from '../../metarpheus/model-ts';
+import { TransitionFunction } from 'state';
+
+type MapProps = {
+  transition: TransitionFunction,
+  getProducts: Array<Book>
+};
+
+export default container(Products)({
+  queries: ['getProducts'],
+  isReady: () => true,
```

Comment:
Qua probabilmente avrà senso togliere `isReady: () => true` e gestire il loading nel componente, magari in un'altra PR